presto-docs/src/main/sphinx/release/release-325.rst

```diff
@@ -2,6 +2,9 @@
 Release 325
 ===========
 
+* Created: 2019-11-14
```

Comment:
can we automate this? please :)

Reply:
Not really easily .. since release process is done by Maven .. so Maven would have to know the document to edit and add it. Imho its not worth the complexity when we already write the release notes themselves manually. If we automate .. it should be the whole release notes creation. However .. thats a LOT of work.